packages/generator-converters/src/wirePropertyV2ToSdkPropertyDefinition.ts

```diff
@@ -25,7 +25,7 @@ import type {
 } from "@osdk/gateway/types";
 
 export function wirePropertyV2ToSdkPropertyDefinition(
-  input: PropertyV2 | SharedPropertyType,
+  input: (PropertyV2 | SharedPropertyType) & { nullable?: boolean },
   isNullable: boolean = true,
```

Comment:
Will we eventually be able to remove the isNullable prop entirely, or are there always cases where we need to specify nullability for certain properties

Reply:
I'd imagine so eventually once the backend gives us all the necessary info we could. If you look through our repo for usages of this function, you'll see that either we pass in 

1. blanket `True` for nullable
2. `!(v2 && objectTypeWithLink.objectType.primaryKey === key),` which basically just says if this isn't a primary key, then it could potentially be null

But, once we can expose info on the backend in the property definitions, we can look at each input property and see if its set to nullable or not without just assuming everything is nullable on our end.

Reply:
Exactly. So with this hack I can at least upgrade really old internal osdk 2.0 usages to use the cli generation to:
1. Generate the code + fetch the ontology.json
2. Run a script to force the properties to `nullable: false` which they were effectively doing by hand
3. Re generate the code without fetching

Otherwise I have to litter their code base with `obj.prop!`. At the end of the day its the EXACT same net effect (we are ignoring the `| undefined` but I don't have to put in a massive delta.